tests/Agent/UnitTests/Core.UnitTest/Spans/SpanStreamingServiceTests.cs

```diff
@@ -717,6 +720,8 @@ public void ErrorsWithTheRequestStream_ShouldShutdownResponseStream()
 
         [Test]
         [Ignore("This test is flickering in our CI", Until = "2022-06-01 00:00:00Z")]
```

Comment:
If we're confident this test will now pass reliably, let's remove the Ignore attribute.

Reply:
They have passed twice in CI now. I'll remove this and see if we can get a 3rd pass :)